src/Parser/EnhancementShaman/CombatLogParser.js

```diff
@@ -0,0 +1,336 @@
+	}
+
+	if(hasHailstorm && frostbrandUptime < .85) {
+		results.addIssue({
+        issue: `Your Frostbrand uptime of ${formatPercentage(frostbrandUptime)}% is below 85%`,
+        icon: SPELLS.FROSTBRAND.icon,
+        importance: getIssueImportance(frostbrandUptime, 0.35, 0.4, true),
+      });
+	} else if(!hasHailstorm && frostbrandUptime > 0) {
+    results.addIssue({
+        issue: `Casting Frostbrand without Hailstorm talent is not recommended`,
+        icon: SPELLS.FROSTBRAND.icon,
+        importance: ISSUE_IMPORTANCE.MAJOR,
+      });
+    }
+
+    if(hasLandslide && landslideUptime < .85) {
+      results.addIssue({
+          issue: `Your Landslide uptime from Rockbiter of ${formatPercentage(landslideUptime)}% is below 85%`,
+          icon: SPELLS.LANDSLIDE_BUFF.icon,
+          importance: getIssueImportance(landslideUptime, 0.35, 0.4, true),
+        });
+    }
+
+    if(hasFuryOfAir && furyofairUptime < .95) {
+      results.addIssue({
+          issue: `Your Fury of Air ${formatPercentage(furyofairUptime)}% uptime is below 95%`,
+          icon: SPELLS.FURY_OF_AIR.icon,
+          importance: getIssueImportance(furyofairUptime, 0.35, 0.4, true),
+        });
+    }
+
+    const castEfficiencyCategories = SPELL_CATEGORY;
+    const castEfficiency = getCastEfficiency(CPM_ABILITIES, this);
+    castEfficiency.forEach((cpm) => {
+        if (cpm.canBeImproved && !cpm.ability.noSuggestion) {
+            results.addIssue({
+                issue: <span>Try to cast <SpellLink id={cpm.ability.spell.id} /> more often ({cpm.casts}/{cpm.maxCasts} casts: {Math.round(cpm.castEfficiency * 100)}% cast efficiency). {cpm.ability.extraSuggestion || ''}</span>,
+                icon: cpm.ability.spell.icon,
+                importance: cpm.ability.importance || getIssueImportance(cpm.castEfficiency, cpm.recommendedCastEfficiency - 0.05, cpm.recommendedCastEfficiency - 0.15),
+            });
+        }
+    });
+
+    //setup notworthyitems
+
+    results.statistics = [
+      <StatisticBox
+        icon={ <Icon icon="class_shaman" alt="DPS stats" /> }
+        value={`${formatNumber(this.totalDamageDone / this.fightDuration * 1000)} DPS`}
+        label={(
+          <dfn data-tip={`The total damage done recorded was ${formatThousands(this.totalDamageDone)}.`}>
+            Damage done
+          </dfn>
+        )}
+      />,
+	  <StatisticBox
+        icon={<Icon icon="spell_mage_altertime" alt="Dead GCD time" />}
+        value={`${formatPercentage(deadTimePercentage)} %`}
+        label={(
+          <dfn data-tip="Dead GCD time is available casting time not used. This can be caused by latency, cast interrupting, not casting anything (e.g. due to movement/stunned), etc.">
+            Dead GCD time
+          </dfn>
+        )}
+      />,
+      <StatisticBox
+          icon={<Icon icon="spell_nature_shamanrage" alt="Core Stats" />}
+          value={(
+              <span>
+                  {`${formatPercentage(masteryPercent)} %`}{' '}
```

Comment:
This could be written as `{formatPercentage(masteryPercent)} %{' '}`, and you could even write it as a one liner; `{formatPercentage(masteryPercent)} % M<br />`